modules/oc-by-example-content.adoc

```diff
@@ -1857,7 +1857,10 @@ Log in to a server
   oc login localhost:8443 --web --callback-port 8280
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email,profile --callback-port=8080
+
+  # Log in with an external OIDC if the external OIDC certificate is not publically trusted
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>
```

Comment:
Hi, the updates are good. But one previous comment https://github.com/openshift/openshift-docs/pull/74184#discussion_r1556795939 seems not yet resolved. Could you help check?

Reply:
@xingxingxia - I have added the explanation of the arguments for this particular command. I hope that helps. Thanks!